source/material_model/latent_heat.cc

```diff
@@ -159,6 +152,11 @@ namespace aspect
             // in the end, all the influences are added up
```

Comment:
Can you add another sentence or so explaining the difference between the Boussinesq and non-Boussinesq approximations. Something like "For non-Boussinesq approximations, the compositional, pressure and phase effects on the density are added to the reference density, and then the thermal effects are applied to that modified density. For the Boussinesq approximation, the temperature effects are first applied to the reference density, and then the composition and phase effects are added on (Boussinesq neglects pressure effects on the density)".